python/dgl/nn/pytorch/conv/twirlsconv.py

```diff
@@ -0,0 +1,660 @@
+"""Torch modules for TWIRLS"""
+# pylint: disable=invalid-name, useless-super-delegation, no-member
```

Comment:
Ideally we shouldn't have these options.  What was the concrete error message if you remove it?

Reply:
'invalid-name' is for those variable names with only one capital letter (like T), to be consistent with the paper.

'useless-super-delegation' is for some classes that are conceptually subclass of nn.Module but actually have no parameters (some original classes in pytorch also defined this way, like nn.Dropout).

if remove 'no-member', it reports some functions in dgl.nn.function don't exist, but atually they do. BTW it seems all other implementations in dgl.nn.pytorch.conv disabled this option.